src/test/java/org/junitpioneer/jupiter/ReportEntryExtensionTests.java

```diff
@@ -338,7 +338,7 @@ void conditional_doesNotLogOnDisabled() {
 
 	@Nested
 	@DisplayName("with parameterized tests")
-	class ParameterEntriesTests {
+	class PropertyEntriesTests {
```

Comment:
Why did you rename this? It looks like it's indeed related to parameterized tests.

Reply:
IntelliJ hickup during some other renaming. I'll revert it